go/teams/service_helper.go

```diff
@@ -111,7 +111,7 @@ func userVersionsToDetails(ctx context.Context, g *libkb.GlobalContext, uvs []ke
 	for i, uv := range uvs {
 		uids[i] = uv.Uid
 	}
-	packages, err := g.UIDMapper.MapUIDsToUsernamePackages(ctx, g, uids, 10*time.Minute, 0, true)
+	packages, err := g.UIDMapper.MapUIDsToUsernamePackages(ctx, g, uids, 10*time.Minute, 0, forceRepoll)
```

Comment:
or was this intentional?

Reply:
good question... I have another PR that removes `forceRepoll` from `Details` completely because it wasn't used

Reply:
ah but this is not forceRepoll but `forceNetworkForFullNames bool` which is something to get full names recent.

Reply:
ah you're right sorry mis read that param initially